app/routes/comment.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express')
+const router = express.Router()
+const commentController = require('../controllers/comment')
+
+router.post(
+  '/',
+  commentController.comment
+)
+
+router.delete(
+  '/:id',
+  commentController.delete
+)
+
+router.patch(
+  '/:id',
+  commentController.update
+)
+
+router.get(
+  '/:id',
+  commentController.getCommentByPost
+)
+
+router.put(
+  '/upvote/:id/:userId',
+  commentController.upvote
+)
+
```

Comment:
@rak-shit Instead of making two api for upvote and downvote
you should make a single API for both

Reply:
Yeah I got what you are saying but I feel it would hardly decrease the number of lines of code. Moreover, I did it this way because it would be easy to perceive for the reader. Though I will try doing what you suggested, if that makes a significant difference in the number of lines I will adopt it. :)

Reply:
it will indeed reduce the code to almost half because now you don't have to write the code common in both the API as you can see i am not going in depth but can have a look